### PR TITLE
Chore: Add Confluent Cloud Quickstart

### PR DESCRIPTION
**Pull request by nr-jbiggley**
# Summary

A concise description of the changes being introduced. If you are submitting a PR for a
new quickstart please review the pre merge checklist section to validate this PR is ready for review.

## Pre merge checklist

- [ Y ] Did you check you NRQL syntax? - Does it work?
- [ Y ] Did you check your dashboard image quality? -  Do they look good?
- [ N/A ] Did you check that your alerts actually work?
- [ Y ] Did you include an InstallPlan and Documentation reference?
- [ Y ] Did you check your descriptive content for voice, tone, spelling and grammar errors?
- [ Y ] Did you attach images of your dashboards to the PR so we can see them working?

### Screenshots
confluentcloud01
confluentcloud02
confluentcloud03

### CODE REVIEW

File: install/third-party/confluent-cloud/install.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+description: |
+  Observability for Confluent Cloud's fully managed service for Apache Kafka on New Relic delivered using the OpenTelemetry Collector
+target:
+  type: integration
+  destination: cloud
+
+install:
+  mode: link
+  destination:
+    url: https://docs.newrelic.com/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/collector/collector-configuration-examples/opentelemetry-collector-kafka-confluentcloud
```
Comment (josephgregoryii) [inline install/third-party/confluent-cloud/install.yml:13]: Looks like this link doesn't exist yet - can we update this to a working URL?

File: quickstarts/confluent-cloud/config.yml [hunk 2]
```diff
@@ -0,0 +1,61 @@
+id:
```
Comment (josephgregoryii) [inline quickstarts/confluent-cloud/config.yml:1]: We generate the `id` field through the merge process - We can get rid of this line

File: quickstarts/confluent-cloud/config.yml [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+  - newrelic partner
+
+# Reference to install plans located under /install directory
+# Allows us to construct reusable "install plans" and just use their ID in the quickstart config
+installPlans:
+  - third-party-confluent-cloud
+
+documentation:
+  - name: Deploy an OpenTelemetry Collector for Confluent Cloud & Kafka metrics collection
+    url: https://docs.newrelic.com/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/collector/collector-configuration-examples/opentelemetry-collector-kafka-confluentcloud
```
Comment (josephgregoryii) [inline quickstarts/confluent-cloud/config.yml:56]: Same issue with this URL here

**Review by josephgregoryii — COMMENTED** [review]
Looks good! Just a couple comments

**Comment by lazyplatypus** [comment]
waiting on https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/14167

**Comment by abeach-nr** [comment]
I will need to update the dashboard json file, as the attribute names will changes as a result of feedback from my pr to the collector-contrib repo.

File: install/third-party/confluent-cloud/install.yml (hunk 1 above)
Comment (nr-jbiggley) [reply install/third-party/confluent-cloud/install.yml:13]: @josephgregoryii We're just waiting on the docs team to publish the updates and then the URL will be live. I'll post here when it is done.

File: quickstarts/confluent-cloud/config.yml (hunk 2 above)
Comment (nr-jbiggley) [reply quickstarts/confluent-cloud/config.yml]: Removed.

File: quickstarts/confluent-cloud/config.yml (hunk 3 above)
Comment (nr-jbiggley) [reply quickstarts/confluent-cloud/config.yml:56]: Waiting on docs site PR to be merged and deployed. Will post when it is live.

File: install/third-party/confluent-cloud/install.yml (hunk 1 above)
Comment (nr-jbiggley) [reply install/third-party/confluent-cloud/install.yml:13]: @josephgregoryii The docs are live! https://docs.newrelic.com/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/collector/collector-configuration-examples/opentelemetry-collector-kafka-confluentcloud/

File: quickstarts/confluent-cloud/config.yml (hunk 3 above)
Comment (nr-jbiggley) [reply quickstarts/confluent-cloud/config.yml:56]: @josephgregoryii Docs are live! https://docs.newrelic.com/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/collector/collector-configuration-examples/opentelemetry-collector-kafka-confluentcloud/

**Review by mickeyryan42 — APPROVED** [review]

**Pull request merged**